packages/amplify_api_plugin_interface/lib/src/graphql/graphql_request.dart

```diff
@@ -30,7 +30,7 @@ class GraphQLRequest<T> {
   Map<String, dynamic> serializeAsMap() => <String, dynamic>{
         'document': document,
         'variables': variables,
-        'cancelToken': cancelToken,
+        'cancelToken': id,
```

Comment:
*nit/question:* Would it make sense to update this to `'id': id,` (and then make the required updates on the platform side)?

Reply:
This would affect the REST API as well since they share some functions on the platform side. Happy to make the change, but initially held off because of that.